### PR TITLE
#113: Info on data freshness

### PR DESCRIPTION
**Pull request by dsuren1**
## Description
This PR adds commit to display data freshness of the cadastrapp and land plan and along with build information

**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/geosolutions-it/MapStore2/blob/master/CONTRIBUTING.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x", remove the others)
 - [x] Enhancement

##Issue

**What is the current behavior?**
#113

**What is the new behavior?**
The Cadastrapp plugin welcome area will display the dates of the cadatrapp and land plan from configuration along with the build information

## Breaking change
**Does this PR introduce a breaking change?** (check one with "x", remove the other)
 - [ ] Yes, and I documented them in migration notes
 - [x] No

### CODE REVIEW

File: assets/translations/data.it-IT.json [hunk 1]
```diff
@@ -3,6 +3,8 @@
     "messages": {
         "cadastrapp": {
             "title": "Cadastrapp",
+            "dateValiditeEDIGEO": "Versione planimetria catastale",
+            "dateValiditeMajic": "Versione dati terra Land",
```
Comment (offtherailz) [inline assets/translations/data.it-IT.json:7]:
```suggestion
            "dateValiditeMajic": "Versione dati terra",
```

**Review by offtherailz — APPROVED** [review]
I pushed my changes. Please check if ok @dsuren1 image

**Comment by dsuren1** [comment]
Looks fine. Thanks @offtherailz :)

**Comment by MaelREBOUX** [comment]
Looks great !

**Comment by MaelREBOUX** [comment]
installed here (tests in progress)

but I note a "truc bizarre". seem there is an offset between git tag and title of the release ?

image

**Comment by MaelREBOUX** [comment]
I can confirm the wrong title.
I let myself correct the title.

**Pull request merged**